src/subscription_manager/managercli.py

```diff
@@ -2298,6 +2300,28 @@ def _do_command(self):
         if self.options.consumed:
             self.print_consumed(service_level=self.options.service_level, filter_string=self.options.filter_string, pid_only=self.options.pid_only)
 
+    def translate_filter_string(self, filter_string=None):
+        if not filter_string:
+            return None
+        return fnmatch.translate(filter_string)
+
+    def highlight_by_filter_string(self, body, filter_string=None):
```

Comment:
These methods don't need to be methods on this class, maybe move to their own module or class? Would make it easier to test if nothing else.